dbt_semantic_interfaces/call_parameter_sets.py

```diff
@@ -16,15 +16,16 @@
 
 @dataclass(frozen=True)
 class DimensionCallParameterSet:
-    """When 'Dimension(...)' is used in the Jinja template of the where filter, the parameters to that call."""
+    """When 'Dimension(...)' is used in a Jinja template, the parameters to that call."""
```

Comment:
Removed specificity to where filter here, since Jinja parsing is used elsewhere now, too.